File: poc/pltests/plcrashtest/SampleCrashTestTV/ViewController.m
//
//  ViewController.m
//  SampleBRTCrashTestTV
//
//  Created by Nallamothu Tharun Kumar on 8/9/19.
//  Copyright © 2019 AT&T CDO. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)testButtonAction:(id)sender {
    NSLog(@"Warning: test app will crash");
    __builtin_trap();
    // crash
}

@end

File: TNHelperSwift/TNHelperSwift/ViewController.swift
//
//  ViewController.swift
//  TNHelperSwift
//
//  Created by Nallamothu Tharun Kumar on 6/24/19.
//  Copyright © 2019 ATT CDO. All rights reserved.
//

import UIKit

class ViewController: UIViewController, Storyboarded {
    weak var coordinator: AppCoordinator?
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    @IBAction func byuapped(_ sender: Any) {
        coordinator?.buySub()
    }
    
    @IBAction func loginTapped(_ sender: Any) {
        coordinator?.loginSub()
    }
}

